app/controllers/emancipation_checklists_controller.rb

```diff
@@ -1,11 +1,12 @@
 class EmancipationChecklistsController < ApplicationController
+  include DateHelper
   before_action :require_organization!
   after_action :verify_authorized
 
   def index
     authorize :application, :see_emancipation_checklist?
     org_cases = current_user.casa_org.casa_cases.includes(:assigned_volunteers)
-    @casa_transitioning_cases = policy_scope(org_cases).where(transition_aged_youth: true).includes([:hearing_type, :judge])
+    @casa_transitioning_cases = policy_scope(org_cases).where("birth_month_year_youth < ?", 14.years.ago).includes([:hearing_type, :judge])
```

Comment:
Hi @carolyn-manning! how are you? , there is a good way to do this using infinity range.

```
  @casa_transitioning_cases = policy_scope(org_cases).where(birth_month_year_youth: 14.years.ago..).includes([:hearing_type, :judge])
```